apidef/oas/xtyk_doc.go

```diff
@@ -0,0 +1,262 @@
+package oas
+
+//go:generate go test -run=TestExtractDocFromXTyk . -timeout 7s -x-tyk-dump-doc=schema/x-tyk-doc.json
```

Comment:
can we maintain this as a separate executable instead of a test?

Reply:
resolved since this task is marked a s feature rather than a client tool by team.